CI.hs

```diff
@@ -237,8 +237,10 @@ buildDists ghcFlavor resolver = do
     stack "--no-terminal exec -- strip-locs examples/mini-compile/test/MiniCompileTest.hs"
     stack "--no-terminal exec -- mini-compile examples/mini-compile/test/MiniCompileTest.hs"
 
-#if __GLASGOW_HASKELL__ >= 808 && defined (mingw32_HOST_OS)
-    -- Skip these tests on ghc-8.8.1 (and greater) for now. See
+#if __GLASGOW_HASKELL__ >= 808 && \
```

Comment:
That would skip the test for `8.10.1` as well. Is that what we want?

Reply:
It looks like this should probably be `if __GLASGOW_HASKELL__ < 808 || _GLASGOW_HASKELL_ == 808 && __GLASGOW_HASKELL_PATHLEVEL1__ == 1 …`

Reply:
Yes. That's an error. Thank-you.